api/src/lib.rs

```diff
@@ -563,6 +568,12 @@ pub struct ListInvoiceParams {
     pub label: Option<String>,
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct LspListProtocolsParams {
```

Comment:
I've been adding new API endpoints to the OpenAPI spec.yaml. The query params won't work properly as you saw in the other PR but at the response type will. I'm trying to not add anything new to this crate and gradually delete it.